documents/2-RTB/Documentazione_Esterna/Glossario.typ

```diff
@@ -0,0 +1,425 @@
+utilizzati in diversi contesti, come il monitoraggio delle prestazioni aziendali, la visualizzazione dei dati di analisi
+e la gestione dei progetti. Nel contesto dello sviluppo software, si riferisce a un'interfaccia grafica o a
+un'applicazione web che fornisce una panoramica visiva delle metriche e delle statistiche rilevanti per il controllo
+della qualità del software. Questo cruscotto di solito raccoglie dati da varie fonti, come sistemi di gestione del
+versionamento del codice, strumenti di test automatizzati, sistemi di monitoraggio delle prestazioni e altri strumenti
+di analisi. Il suo obiettivo principale è quello di fornire agli sviluppatori, ai tester e ai responsabili un modo
+rapido ed efficace per valutare lo stato del progetto, identificare eventuali problemi o anomalie e prendere decisioni
+informate sulle azioni da intraprendere per migliorare la qualità del software
+
+#pagebreak()
+
+= #LTG("D")
+
+== #LTG("Dev team")
+
+In Scrum, il Development Team si riferisce all'insieme dei membri che svolge un ruolo nello sviluppo e nel supporto del
+prodotto e può includere ricercatori, architetti, designer, programmatori, e molti altri
+
+== #LTG("Diagramma dei casi d'uso")
+
+Un diagramma di casi d'uso è uno strumento grafico utilizzato nella modellazione UML (Unified Modeling Language) per
+rappresentare le interazioni tra gli attori (utenti o sistemi esterni) e un sistema software. Serve a descrivere le
+funzionalità del sistema dal punto di vista dell'utente e a documentare i requisiti funzionali in modo chiaro e visivo.
+
+#pagebreak()
+
+= #LTG("E")
+#pagebreak()
+
+= #LTG("F")
+#pagebreak()
+
+= #LTG("G")
+
+== #LTG("GitHub")
+
+GitHub è una piattaforma di sviluppo collaborativo basata su Git, un sistema di controllo delle versioni distribuito. È
+ampiamente utilizzato da sviluppatori di software e team di sviluppo per gestire progetti, tenere traccia delle
+modifiche al codice sorgente, coordinare il lavoro tra i membri del team e facilitare la collaborazione su larga scala
+
+== #LTG("Glossario")
+
+Elenco strutturato di termini tecnici o specializzati, ognuno corredato dalla propria definizione o spiegazione. Questo
+strumento aiuta a migliorare la comunicazione tra le varie parti coinvolte in un progetto, riducendo le ambiguità e
+garantendo una comprensione condivisa dei termini utilizzati in un determinato contesto
+
+== #LTG("GUI (Graphical User Interface)")
+
+Una GUI (acronimo di Graphical User Interface, in italiano Interfaccia Grafica Utente) è un'interfaccia che consente
+agli utenti di interagire con un sistema informatico o un software attraverso elementi visivi e grafici, anziché
+utilizzare solo comandi testuali o linee di comando.
+
+#pagebreak()
+
+= #LTG("H")
+#pagebreak()
+
+= #LTG("I")
+#pagebreak()
+
+= #LTG("J")
+
+== #LTG("Jira")
+
+Jira è uno strumento software sviluppato da Atlassian, utilizzato per la gestione di progetti, il monitoraggio dei
```

Comment:
Magari si potrebbe aggiungere e mettere piu in rilievo l'importanza di Jira come servizio di ticketing (visto che azzurro ce l'ha marcata molto). Un esempio:
"Jira è uno strumento software sviluppato da Atlassian, utilizzato per la gestione di progetti, il monito
raggio dei problemi e il controllo dello sviluppo agile. È particolarmente popolare tra i team di sviluppo
software, specie grazie al servizio di ticketing che offre, ma è anche ampiamente utilizzato in altri ambiti aziendali"

Reply:
(la modifica del mio esempio è quindi a riga 149-150)